_events/2019/sydney.md

```diff
@@ -32,20 +32,24 @@ host: aaron_powell
 speakers:
   amy_kapernick:
     talk: "So You Think You Know HTML?"
+    place: "champion"
```

Comment:
She hasn't won yet. This is for tagging the  finalists after the event.

Reply:
Oh right, I thought it was for tagging of previous events.

Can we tag people who've previously won?

Reply:
Should be done already. If you click on the speaker name, it shows you their history. Like this:
https://pubconf.io/speakers/amy_kapernick/

Reply:
Ah cool, so it is.

I should actually look at how the codebase works 😂